src/pages/index.js

```diff
@@ -84,9 +84,8 @@ function IndexPage() {
                 <h2>#VetsWhoCode</h2>
               </div>
               <p>
-                #VetsWhoCode is a veteran-led and operated 501(c)(3) charitable non-profit that
-                focuses on training veterans in programming and open source development with the
-                purpose of starting careers in the technology industry.
+              VetsWhoCode is an inclusive, open-source, veteran-led and operated 501(c)(3) charitable public non-profit organization.
```

Comment:
Strike "charitable"

Reply:
Helps with the SEO as well. https://apps.irs.gov/pub/epostcard/dl/FinalLetter_86-2122804_VETSWHOCODEINC_05272021_00.tif